ViAn/GUI/projectwidget.cpp

```diff
@@ -132,22 +132,27 @@ void ProjectWidget::start_analysis(VideoProject* vid_proj, AnalysisSettings* set
  * @param tag
  * Adds a tag 'tag' under vid_proj
  */
-void ProjectWidget::add_basic_analysis(VideoProject* vid_proj, BasicAnalysis* tag) {
+void ProjectWidget::add_basic_analysis(VideoProject* vid_proj, Tag* tag) {
```

Comment:
We should probably change the function name so it matches what is added. `add_tag` perhaps

Reply:
Once upon a time it had that name as seen in the comment